staging/src/k8s.io/code-generator/cmd/validation-gen/validation.go

```diff
@@ -943,6 +960,37 @@ func (g *genValidations) emitValidationForChild(c *generator.Context, thisChild
 				case types.Struct:
 					// Call the type's validation function.
 					g.emitCallToOtherTypeFunc(c, fld.node, bufsw)
+					for _, subchild := range fld.inner {
+						if len(subchild.name) == 0 {
+							klog.Fatalf("missing child name for field in %v", thisNode)
```

Comment:
change to panic(Sprintf(...)) like other examples, so we get the trace

Reply:
done